README.md

```diff
@@ -148,25 +146,38 @@ We support following browsers:
 - MS Edge (latest version)
 - IE11
 
-#### CLI commands
+### CLI commands
 
-**Run LESS compiler watching for local dev**
 
-	cd repository-data/webfiles
-	mvn lesscss:compile -D lesscss.watch=true
+**Start frontend build process for local dev**
 
-**Run SASS compiler watching for local dev**
+This will:
 
-	cd repository-data/webfiles
-	mvn com.github.warmuuh:libsass-maven-plugin:0.2.8-libsass_3.4.4:watch
+- Install NPM dependencies
+- Run a gulp task which:
+    - starts the SCSS compilation into CSS
+    - starts the JS compilation via Webpack
+    - starts Browsersync to stream in frontend changes
 
-**Run the local dev server (will throw a lot of Splunk errors)**
+This requires a local installation of [Node.js](https://nodejs.org).
 
-  mvn clean verify && mvn -Pcargo.run -Drepo.path=storage
+Note: This is aliased to `make frontend`
```

Comment:
Can you add a jump link to: 'About the Frontend Build process' more information link.

Reply:
Or to docs/what-if/work-on-ui.md